EnglishFilter/sections/specific.txt

```diff
@@ -14958,3 +14958,75 @@ adguard.com##.hello_from_adguard_apps
 !+ PLATFORM(windows, mac, android, ios) NOT_OPTIMIZED
 adguard-dns.com##.hello_from_adguard_apps
 !
+www.indiatoday.in##.story-social-share-fix
+www.indiatoday.in##.amp-izooto-sub
+www.thehindu.com##.i-amphtml-unresolved
+indianexpress.com##.breaking-scrollbx
+indianexpress.com##.js-btm-nav
+indianexpress.com##.i-amphtml-unresolved
+indianexpress.com##.top-ads-mobile
+indianexpress.com##.pop_up_box
+indianexpress.com##.app-btn__link
+indianexpress.com##.iframe-lazy
+amp.scroll.in##.amp-ad-wrapper
+www.espncricinfo.com##.Ad__Wrapper
+www.espncricinfo.com##.i-amphtml-unresolved
+www.thebetterindia.com##.adp-post-ad-item.adp-wrapper
+m.timesofindia.com##.location_detect_boxn1
+m.timesofindia.com##.floatingwhtsappicon
+m.timesofindia.com##.topadCLSPlaceholder
+www.deccanherald.com##.mtop-m-20.section-container
+www.deccanherald.com##.amp-ad
+www.gqindia.com##.i-amphtml-unresolved
+www.ndtv.com##.story_ft-nav
+www.analyticsinsight.net##.mfp-content
+www.analyticsinsight.net##.mfp-ready.mfp-auto-cursor.mfp-wrap
+www.analyticsinsight.net##.mfp-ready.mfp-bg
+www.designboom.com###db-cookie-notice
+www.designboom.com##.open.nlbox-popup-wrapper
+m.economictimes.com###appinstall_bnr
+m.economictimes.com###floatingAppButton
+www.dkoding.in##.mrf-collapseFooter
+zeenews.india.com###warning-message
+zeenews.india.com##.subscribe
+zeenews.india.com##.i-amphtml-unresolved
+www.ndtv.com###___ndtvpushdiv
+www.ndtv.com##.tbl-floating-unit
+indianexpress.com##.top-ads-amp
+www.livemint.com##.openinApp
+www.pinkvilla.com##.ad-heder-320-50
+www.pinkvilla.com##.amp-article-footer.sharing-section
+www.pinkvilla.com##.fe.bodyads
+www.pinkvilla.com##.taboola-ads
+auto.hindustantimes.com##.amp-adsheight.adHolder
+www.archdaily.com##.kth-toast--bottom.kth-toast
+www.archdaily.com###js-afd-nav
+kalingatv.com##.header-inner
+www.newsnext.live##.onesignal-popup_wrapper
+www.newsnext.live##.ss-ic.s_stk
+m.timesofindia.com##.notification_wrapper
+m.timesofindia.com##.floatingoneapp
+m.timesofindia.com##.scroll_bottom
+m.timesofindia.com##.menu_bottom
+www.architecturaldigest.in##.i-amphtml-unresolved
+tech.hindustantimes.com##.adsHeight300x250
+tech.hindustantimes.com##.FirstAd
+tech.hindustantimes.com##.bottomStickyAds
+m.economictimes.com##.amazon_affi
+www.moneycontrol.com###download_app_banner_btn
+www.moneycontrol.com##.pushbtn.align-center-button
+www.moneycontrol.com##.i-amphtml-unresolved
+timesofindia.indiatimes.com###amp_float_whtsapp
+www.dnaindia.com##.i-amphtml-unresolved
+www.koimoi.com##.i-amphtml-unresolved
+science.thewire.in###wpfront-notification-bar-table
+science.thewire.in###scroll_to_top
+yourstory.com###login-popup
+www.digitalcameraworld.com###taboola-mobile-below-article-thumbnails
+www.bloombergquint.com##amp-subscriptions-dialog
+www.bloombergquint.com##.social-share-icons
+www.gizmochina.com##.post_td_gallery
+www.zoomtventertainment.com##.i-amphtml-unresolved
+www.ndtv.com###jiosaavn-widget
```

Comment:
Checked the first random rule - this is not an ad, this is annoyance.
As I see these rules are not sorted by categories. We cannot merge it.